lib/logstash/filters/split.rb

```diff
@@ -18,7 +20,7 @@ class LogStash::Filters::Split < LogStash::Filters::Base
   # string.
   config :terminator, :validate => :string, :default => "\n"
 
-  # The field which value is split by the terminator
+  # The field which value is split by the terminator.  Can be a multiline message or the ID of an array.  Nested arrays are referenced like: "[object_id][array_id]"
```

Comment:
It's okay to split this into multiple lines, as long as it starts with a `#`.  Asciidoc makes it all good.

Would you please make sure the lines aren't longer than around 80 characters?  It's kind of a coding standard we try to maintain for readability.

Otherwise it LGTM™.